samples/ComponentExamples/src/utils/utils.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { CommunicationIdentityClient, CommunicationUserToken } from '@azure/communication-identity';
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+import { ChatClient } from '@azure/communication-chat';
+
+export type IdentityType = {
+  userId: string;
+  token: string;
+  endpointUrl: string;
+  displayName: string;
+  threadId: string;
+};
+
+const CONNECTION_STRING = require('../../appsettings.json')['ResourceConnectionString'];
```

Comment:
This appsettings.json is to keep the example in a working status before I make the automation test PR (which help me split things into smaller PR) - will replace this part and use URLParam + Server request in next automtaion test PR